git_machete/github.py

```diff
@@ -29,20 +28,21 @@
 
 
 class GitHubPullRequest(object):
-    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str) -> None:
+    def __init__(self, number: int, user: str, base: str, head: str, html_url: str, state: str, repo: Dict[str, Any]) -> None:
```

Comment:
Hmmm what are the other fields in `repo` other than `full_name`? do we need other than `full_name`?

Reply:
Maybe a dedicated structure for the repo would be a better choice? Coz `Dict` is basically un-mypy-checkable :/

Reply:
yes, also repository url is taken from there. Maybe extract them in `__parse_pr_json` to string as well